MergeCalendarsTogether.gs

```diff
@@ -11,6 +11,8 @@ const SYNC_DAYS_IN_FUTURE = 30;
 // Set this to "false" when your happy with the debug output!
 const DEBUG_ONLY = true;
 
+const VERBOSE_LOGGING = false;
```

Comment:
I added a line I found too useful for testing, but was messing with running unit tests given our limited test runner solution.